ansible/vars/topo_dualtor-64.yml

```diff
@@ -58,7 +58,7 @@ topology:
     - 0.62@62,1.62@62
     - 0.63@63,1.63@63
   disabled_host_interfaces:
-    - 0.2@2,1.2@2
+    - 0.2@0,1.2@0
```

Comment:
I am afraid that this will cause the DUT port connected to "eth0" of PTF in admin down status. Can you double confirm?

Reply:
Thanks Xin. Yes, the change is intentional. Although the port is admin down, the `eth0` will be present in the ptf container. So the test cases can read MAC address from it.